Doc/library/hashlib.rst

```diff
@@ -284,7 +284,7 @@ a file or file-like object.
    Example:
 
       >>> import io, hashlib, hmac
-      >>> with open(hashlib.__file__, "rb") as f:
+      >>> with open("path/to/file", "rb") as f:
```

Comment:
the previous code worked because it passes doctest.  It probably runs from the repo root, what happens if you use "Doc/library/hashlib.rst" here?

Reply:
Oh, now its even _cleverer_ in that this was hand-picked because we know its gonna pass doctest? 🤦

Reply:
```suggestion
      >>> with open("Doc/library/hashlib.rst", "rb") as f:
```

Reply:
```
    FileNotFoundError: [Errno 2] No such file or directory: 'Doc/library/hashlib.rst'
```

https://github.com/python/cpython/actions/runs/15196498586/job/42741793385?pr=134540#step:8:405